samples/bot-conversation/python/.vscode/tasks.json

```diff
@@ -0,0 +1,79 @@
+// This file is automatically generated by Teams Toolkit.
+// The teamsfx tasks defined in this file require Teams Toolkit version >= 5.0.0.
+// See https://aka.ms/teamsfx-tasks for details on how to customize each task.
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Start Teams App Locally",
```

Comment:
We didn't start the app here, may change to `Prepare Teams App Resources`

Reply:
done